src/sqe.rs

```diff
@@ -347,6 +383,33 @@ impl<'a> SubmissionQueueEvent<'a> {
 unsafe impl<'a> Send for SubmissionQueueEvent<'a> { }
 unsafe impl<'a> Sync for SubmissionQueueEvent<'a> { }
 
+pub struct AcceptParams {
```

Comment:
I'd propose we call this type `SockAddrStorage`, the same name as the low level type but in Rust's casing style.

Reply:
Agreed.